Motorcycle.js

```diff
@@ -1,6 +1,21 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+
+function Motorcycle(make, model, year, color, seats, passengers) {
+	Car.call(this, make, model, year, color, seats, passengers)
+	this.seats = 2;
```

Comment:
Here, you could also use an if statement to override the number of seats input if they are greater than 2:
```
if (this.seats > 2) {
    this.seats = 2;
}